src/api/app/components/request_decision_component.rb

```diff
@@ -6,7 +6,13 @@ def initialize(bs_request:, action:, is_target_maintainer:, package_maintainers:
     @is_target_maintainer = is_target_maintainer
     @action = action
     @package_maintainers = package_maintainers
-    @show_hint = render? && show_project_maintainer_hint
+
+    return unless render? && show_project_maintainer_hint
+
+    @package_maintainers_hint = "Note\n" \
+                                'You are a project maintainer but not a package maintainer. This package ' \
+                                "has #{pluralize(@package_maintainers.size, 'package maintainer')} assigned. Please keep " \
+                                'in mind that also package maintainers would like to review this request.'.freeze
```

Comment:
Why do you do this in `initialize` and not in `confirmation`?

Reply:
Because we reuse this text in two places

Reply:
Because it is shared and used by `other_decision_confirmation` too

Reply:
So why do you skip this under those conditions?

Reply:
`show_project_maintainer_hint` is the condition to render/not render the hint. If the condition is `false` there is no reason to show the hint, and the dialog message of `confirmation` or `other_decision_confirmation` will have/not have the hint accordingly

Reply:
Sounds like `Webui::RequestController#show` implementation details leaking into this component...

Reply:
https://trello.com/c/Yoie6Nvz/251-cleanup-webuirequestcontrollerpreparerequestdata